### PR TITLE
Fix broken master

### PR DESCRIPTION
**Pull request by iocanel**

### CODE REVIEW

File: examples/spring-boot-on-kubernetes-with-jib-custom-registry-example/src/main/resources/application.properties [hunk 1]
```diff
@@ -1,2 +1,3 @@
 server.port=9090
 dekorate.jib.registry=quay.io
+dekorate.docker.enabled=false
```
Comment (aureamunoz) [inline examples/spring-boot-on-kubernetes-with-jib-custom-registry-example/src/main/resources/application.properties:3]: So, we need to disable explicitly conf other than the one we want to use?

**Review by aureamunoz — COMMENTED** [review]
LGTM

File: examples/spring-boot-on-kubernetes-with-jib-custom-registry-example/src/main/resources/application.properties (hunk 1 above)
Comment (iocanel) [reply examples/spring-boot-on-kubernetes-with-jib-custom-registry-example/src/main/resources/application.properties:3]: For now yes.
The next step is to prioritize build services that have been explicitly configured.
For example, if use configures jib and not docker jib gets priority.

Comment (aureamunoz) [reply examples/spring-boot-on-kubernetes-with-jib-custom-registry-example/src/main/resources/application.properties:3]: Got it!

**Review by aureamunoz — APPROVED** [review]

File: examples/spring-boot-on-kubernetes-with-jib-example/src/test/java/io/dekorate/example/SpringBootOnKubernetesTest.java [hunk 2]
```diff
@@ -36,7 +36,6 @@ public void shouldContainDeployment() {
     Deployment d = findFirst(list, Deployment.class).orElseThrow(() -> new IllegalStateException());
     assertNotNull(d);
     Container container = d.getSpec().getTemplate().getSpec().getContainers().get(0);
-    assertTrue(container.getImage().contains("docker.io"));
```
Comment (Sgitario) [inline examples/spring-boot-on-kubernetes-with-jib-example/src/test/java/io/dekorate/example/SpringBootOnKubernetesTest.java:39]: This line was deleted on purpose? Thanks to this assertion this issue was spotted. cc @iocanel

**Pull request merged**